.github/workflows/rust.yml

```diff
@@ -93,14 +93,4 @@ jobs:
             ex=$((ex + 1))
           done
 
-          # Check the policy-controller dockerfiles workflow against the version
```

Comment:
Deleted this since we are now using the dev container. Since that serves as the centralized point for rust tooling, it's unlikely this repo can be on a different version unless the change is made there first.